tests/deltalake_write_notebook.py

```diff
@@ -0,0 +1,11 @@
+# Databricks notebook source
+
+# Instrument for unit tests. This is only executed in local unit tests, not in Databricks.
+if 'dbutils' not in locals():
+    import databricks_test
+    databricks_test.inject_variables()
```

Comment:
yes, I remember now, so we'll need to do this in every notebook under test. I don't understand why the framework was written in this way, there must be a nicer way...